integration/__tests__/cluster-pages.tests.ts

```diff
@@ -417,7 +411,7 @@ const scenarios = [
   {
     expectedSelector: "h5.title",
     parentSidebarItemTestId: "sidebar-item-link-for-user-management",
-    sidebarItemTestId: "sidebar-item-link-for-pod-security-policies",
```

Comment:
minikube does not have PodSecurityPolicy

Reply:
looks like the api reports missing ressources as allowed:

`$ kubectl --context minikube auth can-i list podsecuritypolicies
Warning: the server doesn't have a resource type 'podsecuritypolicies'

yes`